src/macros.rs

```diff
@@ -22,8 +28,8 @@ macro_rules! itry {
     })
 }
 
-/// Unwrap the given Option or return a Ok(Response::new()) with the given modifier. The default
-/// modifier is status::BadRequest.
+/// Unwrap the given `Option` or return a `Ok(Response::new())` with the given
+/// modifier. The default modifier is `status::BadRequest.
```

Comment:
Oops, I only checked `itry` docs in my browser, not `iexpect` :(

Do you want a PR, or will you fix it directly on master?